core/types.go

```diff
@@ -27,6 +32,7 @@ type Authorization struct {
 	ID          string
 	URL         string
 	ExpiresDate time.Time
+	Order       *Order
```

Comment:
I think you meant this to be a list of orders? But AFAICT this isn't used in this PR so should wait.